recipes/monsda/meta.yaml

```diff
@@ -17,32 +17,32 @@ build:
     - monsda = MONSDA.RunMONSDA:main
     - monsda_configure = MONSDA.Configurator:main
   noarch: python
+  run_exports:
+    - {{ pin_subpackage("monsda", max_pin="x") }}
 
 requirements:
   host:
-    - python >=3.9
+    - python >=3.11.4
     - pip
     - setuptools
     - pytest-runner
   run:
-    - biopython>=1.79
-    - grep>=3.4
-    - libmamba>=0.21.2
-    - mamba>=0.17.0
-    - more-itertools>=8.8.0
-    - natsort>=7.1.1
-    - nextflow
-    - numpy>=1.21.0
-    - pandas>=1.4.1
-    - perl>=5.32.1
-    - python>=3.9
-    - scipy>=1.7.0
-    - snakemake-minimal>=6.5.3
-    - snakemake>=6.5.3
-    - versioneer>=0.20
-    - yaml>=0.2.5
-    - zlib>=1.2.11
-    - zstd>=1.5.0
+    - grep >=3.4
+    - conda-libmamba-solver >=23.5.0
+    - isort >=5.10.1
+    - mamba >=1.0.0
+    - more-itertools >=8.12.0
+    - natsort >=8.1.0
+    - nextflow =23.04.1
+    - perl =5.32.1
+    - python =3.11.4
+    - python_abi =3.11
+    - python-dateutil >=2.8.2
+    - python-fastjsonschema >=2.15.3
+    - python-irodsclient >=1.1.3
+    - pyopenssl >=22.0
+    - pyyaml >=6.0
+    - snakemake =7.32.3
```

Comment:
Here as well, a less stringent pin might allow more mix and match with other tools

Reply:
snakemake and nextflow need stringent pinning, unable to test with every minor patch